rules.bzl

```diff
@@ -28,11 +28,17 @@ def _android_ndk_repository_impl(ctx):
         fail("Either the ANDROID_NDK_HOME environment variable or the " +
              "path attribute of android_ndk_repository must be set.")
 
+    is_windows = False
+    tool_extension = ""
```

Comment:
i think a slightly more precise name like "executable_extension" might be better

Reply:
Done